dev/integration_tests/ios_platform_view_tests/lib/main.dart

```diff
@@ -0,0 +1,104 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import 'package:flutter_driver/driver_extension.dart';
+
+void main() {
+  enableFlutterDriverExtension();
+  runApp(MyApp());
+}
+
+/// The main app entrance of the test
+class MyApp extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+      title: 'Flutter Demo',
+      theme: ThemeData(
+        primarySwatch: Colors.blue,
+      ),
+      home: const MyHomePage(title: 'Flutter Demo Home Page'),
+    );
+  }
+}
+
+/// A page with a button in the center.
+///
+/// On press the button, a page with platform view should be pushed into the scene.
+class MyHomePage extends StatefulWidget {
+  const MyHomePage({Key key, this.title}) : super(key: key);
+  final String title;
+
+  @override
+  _MyHomePageState createState() => _MyHomePageState();
+}
+
+class _MyHomePageState extends State<MyHomePage> {
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
+        title: Text(widget.title),
+      ),
+      body: Column(children: <Widget>[
+        FlatButton(
+          key: const ValueKey<String>('platform_view_button'),
+          child: const Text('platform view'),
+          onPressed: () {
+            Navigator.push(
+              context,
+              MaterialPageRoute<PlatformViewPage>(
+                  builder: (BuildContext context) => PlatformViewPage()),
+            );
+          },
+        ),
+        RaisedButton(
```

Comment:
nit: reading the code here I'm not sure why do we need a no-op button here? can you clarify in a comment?

Reply:
We tap this button to pump some frames, in order to un-merge the thread. (In the flutter driver test)
Tapping raised button runs a bunch of animations which pumps many frames.

Reply:
done